volumes/miovision/sql/function-aggregate-volumes_15min_tmc.sql

```diff
@@ -10,62 +10,61 @@ AS $BODY$
 
 BEGIN
 
+WITH zero_padding_movements AS (
+		/*Cross product of legal movement for cars, bikes, and peds and the bins to aggregate*/
+		SELECT m.*, datetime_bin15 
+		FROM miovision_api.intersection_movements_new m
+		CROSS JOIN generate_series(start_date - interval '1 hour', end_date - interval '1 hour 15 minutes', INTERVAL '15 minutes') AS dt(datetime_bin15)
+		WHERE classification_uid IN (1,2,6) 
+		)
+, aggregate_insert AS (
 INSERT INTO miovision_api.volumes_15min_tmc(intersection_uid, datetime_bin, classification_uid, leg, movement_uid, volume)
 
 SELECT 
-m.intersection_uid,
-dt.datetime_bin,
-m.classification_uid,
-m.leg,
-m.movement_uid,
+pad.intersection_uid,
+pad.datetime_bin15,
```

Comment:
This needs to do a `COALESCE(pad.datetime_bin15, datetime_bin(A.datetime_bin, 15) AS datetime_bin`